com.sg.basic.ps.version

```diff
@@ -1 +1 @@
-v1.4.0
\ No newline at end of file
+dev
```

Comment:
Is that on purpose? What is it for?

Reply:
For now, yeah. I'll rebuild the plugin with the release version once it's through QA.